com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -95,6 +102,7 @@ Shader "Universal Render Pipeline/Lit"
             #pragma shader_feature_local_fragment _METALLICSPECGLOSSMAP
             #pragma shader_feature_local_fragment _SMOOTHNESS_TEXTURE_ALBEDO_CHANNEL_A
             #pragma shader_feature_local_fragment _OCCLUSIONMAP
+            #pragma shader_feature_local _DETAIL_MULX2 _DETAIL_SCALED
```

Comment:
Can/Should these be shader_feature_local_fragment like the others there?

Reply:
We need vertex variant too. As detail requires tangetWS interpolator to be calculated.